python/cudf/cudf/pandas/scripts/run-pandas-tests.sh

```diff
@@ -24,7 +24,8 @@ PANDAS_VERSION=$(python -c "import pandas; print(pandas.__version__)")
 
 # tests/io/test_clipboard.py::TestClipboard crashes pytest workers (possibly due to fixture patching clipboard functionality)
 PYTEST_IGNORES="--ignore=tests/io/parser/common/test_read_errors.py \
---ignore=tests/io/test_clipboard.py"
+--ignore=tests/io/test_clipboard.py \
+--ignore=tests/io/test_pickle.py"
```

Comment:
Re-opened the issue related to this: https://github.com/rapidsai/cudf/issues/15459